src/interface/emacs/khoj.el

```diff
@@ -428,40 +434,44 @@ CONFIG is json obtained from Khoj config API."
              (openai (assoc 'openai conversation)))
         (when openai
           ;; Unset the `openai' field in the khoj conversation processor config
-          (message "khoj.el: disable Khoj Chat using OpenAI as your OpenAI API key got removed from config")
+          (message "khoj.el: Disable Chat using OpenAI as your OpenAI API key got removed from config")
           (setcdr conversation (delq openai (cdr conversation)))
```

Comment:
Do you have to do anything to set the new value of `conversation` in `processor`? Or will this transfer automatically?

Reply:
Nevermind, clarified offline.

Reply:
We just set `api-key` field under processor > conversation > openai section to null in this section. The next section adds this updated conversation section to processor